### DISCUSSION TITLE
POV, Mouselook and Opentrack together!

### DISCUSSION THREAD

**Discussion by thbrix** [opentrack/opentrack#1540] [General]
Hi and thanks a lot for your great software! I use it with BMS.

I have one very big wish for you and that is, that I could be able to pause opentrack, and then use mouselook and pov. The reason is, that I need to rest my neck on long flights, and it would be great to only enable headtracking when needed, and use pov and mouse when tracking is paused.

Would that be possible to do?

Best regards
Thomas

**Comment by Rowanor** [comment]
Hi!

I am not 100 percent on this topic but  Falcon BMS registers that you have a head tracker when you first launch the game. You will need to uninstall a head tracker to use mouse look.

**Comment by GO63-samara** [comment]
Well, not really.

To make the mouse work, you need to clear the key in the registry:
`HKEY_CURRENT_USER\Software\NaturalPoint\NATURALPOINT\NPClientLocation\Path`

To make Opentrack work again, you need to restore this key:
`Opentrack` - `freetrack 2.0 Enchanced` - `Settings` - `Locate DLL`
